docs/Examples.md

```diff
@@ -504,6 +504,30 @@ You can also install apps hosted remotely using URLs:
         - "https://example.com/splunk-apps/app3.tgz"
 ```
 
+Also these application configuration parameters can be placed in a `defaults.yml`
+file and use the `defaultsUrlApps` parameter.  The `defaultsUrlApps` parameter
+is specific for applicastion installation and will install the apps in the
+correct instances as per the deployment.
+
+Unlike `defaultsUrl` which is applied at every instance created by the CR, the
+`defaultsUrlApps` will be applied on instances that will **not** get the application
+installed via a bundle push.  earch head and indexer cluster members will not have
+the `defaultsUrlApps` parameter applied.  This means:
+
+ - For standalones the apps will be installed as normal.
+ - For SHC these applications will be installed in the SHC Deployer and push it out
+to search peers via a SH Bundle push.
+ - For IndexerClusters, these applications will only be installed on the CM and
+pushed to the indexers in the cluster via CM Bundle push.
+
+Application and other ansible defaults can be still be installed via `defaultsUrl`
+and both options can be used in conjunction:
```

Comment:
Installing Applications via `defaultsUrl`, while continues to work, is not going to be a recommended practice anymore, right? Especially to avoid what this PR is fixing (the double pod resets).

Reply:
Yes while technically you can still have apps in the `defaultsUrl` this would not be preferred but would be left in for backward compatibility.  The new block would be:

> For application installation the preferred method will be through the `defaultsUrlApps` 
while other ansible defaults can be still be installed via `defaultsUrl`.  For backwards 
compatibility applications could be installed via `defaultsUrl` though this is not 
recommended.  Both options can be used in conjunction: